source/frontend/src/components/common/mapFSM/MapStateMachineContext.tsx

```diff
@@ -223,6 +240,7 @@ export const MapStateMachineProvider: React.FC<React.PropsWithChildren<unknown>>
           { mapVisible: { sideBar: 'sidebarOpen' } },
           { mapVisible: { sideBar: 'selecting' } },
         ].some(state.matches),
+        sideBarType: state.context.sideBarType,
```

Comment:
The current sidebar is the left pane of the map. This would be mixing the filters with that conceptually

Reply:
This is here because we need to expose the type of "thing" being displayed in the "regular" sidebar. That is to comply with the following AC from my story:

- Given advanced search is open, when property marker is clicked, property details left hand sidebar should display and advanced search should stay open
- Given advanced search is open, when user navigates to a file, the advanced search should be closed.

Reply:
So there's logic in MapContainer that listens for what kind of sidebar we have in order to conditionally close the Advance Filter sidebar

Reply:
I like the idea of keeping the current sidebar and the advanced search/layers separate.